test/SConscript

```diff
@@ -3,6 +3,7 @@ import subprocess
 from xml.etree import ElementTree
 from os.path import join as pjoin
 import time
+import sys
```

Comment:
Is this import used in the file?

Reply:
It is - stumbled across this (rarely encountered) glitch when I ran into an exception. Link is here: https://github.com/Cantera/cantera/blob/33a2e87866be22ee08e6b89567e230f3a5627dc6/test/SConscript#L68-L71